ios/FluentUI/Tab Bar/TabBarItem.swift

```diff
@@ -30,6 +30,9 @@ open class TabBarItem: NSObject {
        }
    }
 
+    /// A string that uniquely identifies the element, typically for automation purposes.
+    @objc public var accessibilityIdentifier: String?
```

Comment:
if you create a tabbaritem and the later on change the accessibilityIdenitifer, we still need to notfiy tabbaritemview their accessibilityidentifier has changed